bsm_generator/src/bsm_generator.cpp

```diff
@@ -124,7 +125,11 @@ namespace bsm_generator
     {
         bsm_.header.stamp = ros::Time::now();
         bsm_.core_data.msg_count = worker.getNextMsgCount();
-        bsm_.core_data.id = worker.getMsgId(ros::Time::now());
+        
+        if(bsm_id_rotation_enabled_)
+        {
+            bsm_.core_data.id = worker.getMsgId(ros::Time::now());
```

Comment:
How does this value get set in he non-rotating case? Seems a default value should be generated on node initialization or loaded from a parameter